apps/emqx_connector/src/emqx_connector.erl

```diff
@@ -223,19 +262,21 @@ list() ->
             )
         end,
         [],
-        emqx:get_raw_config([connectors], #{})
+        get_raw_config(Namespace, [connectors], #{})
     ).
 
 lookup(Id) ->
     {Type, Name} = emqx_connector_resource:parse_connector_id(Id),
     lookup(Type, Name).
-
 lookup(Type, Name) ->
-    RawConf = emqx:get_raw_config([connectors, Type, Name], #{}),
-    lookup(Type, Name, RawConf).
-
-lookup(Type, Name, RawConf) ->
-    case emqx_resource:get_instance(emqx_connector_resource:resource_id(Type, Name)) of
+    lookup(_Namespace = undefined, Type, Name).
+lookup(Namespace, Type, Name) ->
+    RawConf = get_raw_config(Namespace, [connectors, Type, Name], #{}),
+    do_lookup(Namespace, Type, Name, RawConf).
```

Comment:
Nit.
```suggestion
    lookup(_Namespace = undefined, Type, Name).

lookup(Namespace, Type, Name) ->
    RawConf = get_raw_config(Namespace, [connectors, Type, Name], #{}),
    do_lookup(Namespace, Type, Name, RawConf).
```

But again, gently suggesting to just make callers aware of namespaces by dropping `lookup/2`.